src/lib/stores/modals.svelte.ts

```diff
@@ -1,10 +1,10 @@
-import {  SvelteMap } from "svelte/reactivity";
+import { Map } from "svelte/reactivity";
```

Comment:
we can just remove this line

Reply:
I dont think this is the right approach.